libvmaf/src/feature/float_adm.c

```diff
@@ -28,6 +28,12 @@
 #include "mem.h"
 #include "picture_copy.h"
 
+#define PROFILE_FLOAT_ADM 1
+
+#if PROFILE_FLOAT_ADM
```

Comment:
You might want to add `&& ! defined(_WIN32)` to this since `sys/time.h` doesn't exist on windows. Although `timeval` is declared in [`winsock2.h`](https://docs.microsoft.com/en-us/windows/win32/api/winsock/ns-winsock-timeval), [`gettimeofday`](https://stackoverflow.com/questions/10905892/equivalent-of-gettimeday-for-windows) is not available on windows

Reply:
Thanks for review, we will update the code and push a new commit